ci/Jenkinsfile

```diff
@@ -1,2 +1,2 @@
 // Warning: This file is automatically synced from https://github.com/ipfs/ci-sync so if you want to change it, please change it there and ask someone to sync all repositories.
-javascript()
+javascript(['nodejs_versions': ['8.11.3']])
```

Comment:
@jacobheun @alanshaw switched to Node.js 8 for now until wrtc has builds for node 10